test/test_jit.py

```diff
@@ -1675,6 +1675,13 @@ def test_script_for_in_range_if_ast(x):
 
         self.assertEqual(test_script_for_in_range_if_ast(*inputs).shape[0], 20)
 
+    def test_script_torch_size(self):
+        @torch.jit.script
+        def func(x):
+            return torch.rand((x.size(0), 1))
+
+        self.assertEqual(func(torch.rand(2, 3)).shape[0], 2)
```

Comment:
It might be better to use self.checkScript to make this more consistent with the other tests?